app/src/main/java/me/saket/dank/ui/submission/SubmissionTitleSpan.kt

```diff
@@ -0,0 +1,16 @@
+package me.saket.dank.ui.submission
+
+import android.text.Spannable
+import me.saket.dank.widgets.TextViewLimitSelection
+
+// This is a no-op span to identify title position in TextView
+class SubmissionTitleSpan {
```

Comment:
Not sure if this will work with spans but perhaps it’s worth changing this to `object` instead of `class`.